### DISCUSSION TITLE
How to create folders?

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1540] [Q&A] [answered]
After I've imported my old notes, I got the previous folder created in another markdown editor and they look like this:
![image](https://user-images.githubusercontent.com/23059969/104822776-f3947c80-57f9-11eb-8e9b-fc71796d13a6.png)

Now I want to create new folders inside Trillium. When I right-click though, I don't see folders nor the folder icon:
![image](https://user-images.githubusercontent.com/23059969/104822788-0eff8780-57fa-11eb-813f-61afce261751.png)

Am I missing something?

**Comment by zadam** [comment, marked as answer]
There isn't "folder" as such. Instead, any note can have children notes. So you can just create "text" note and in that you can create more children notes.
